mozilla_django_oidc/auth.py

```diff
@@ -44,12 +44,22 @@ def __init__(self, *args, **kwargs):
 
         self.UserModel = get_user_model()
 
-    def create_user(self, email, **kwargs):
+    def filter_users_by_claims(self, claims):
+        """Return all users matching the specified email."""
+        email = claims.get('email')
+        if not email:
+            return None
+        return self.UserModel.objects.filter(email=email)
+
+    def create_user(self, claims):
         """Return object for a newly created user account."""
         # bluntly stolen from django-browserid
         # https://github.com/mozilla/django-browserid/blob/master/django_browserid/auth.py
 
         username_algo = import_from_settings('OIDC_USERNAME_ALGO', None)
+        email = claims.get('email')
+        if not email:
+            return None
```

Comment:
This holds true for the `filter_users_by_claims`. The create_user does not return a queryset but and object.